### ISSUE TITLE
Labels for graphs

### ISSUE THREAD

**Issue by stevieflow** [IATI/D-Portal#685]
Now we have https://github.com/IATI/D-Portal/issues/684 in place, it would be good to look at the text used for the graphs - eg:

![d-portal (22)](https://github.com/user-attachments/assets/7cc3fa0c-b892-41fe-8331-63ee84fe1b7b)

> Where does the money come from (IATI all years)

This graphs leads to: https://d-portal.org/ctrack.html?country_code=AO#view=publishers

> The money comes from 67 IATI Reporting Organisations.

![d-portal (23)](https://github.com/user-attachments/assets/91658b10-08ee-4495-9436-2dfbd9f9a8a7)

I'm not too keen on the sentiment of "the money come from" as it's not entirely true.  The *money* may come from different sources -- > eg a ``reporting-org`` may have activities whereby they are _not_ a ``participating-org/@role="1"`` (Funding).  There will be quite a few other examples, too

Before proceeding, just wanted to check with @notshi & @xriss that this statement ("Where does the money come from (IATI all years)") now relates to a breakdown of the relevant ``reporting-org`` ?

Note: this would also be something to look at for the sector and country graphs, at a later point

cc/ @isabelbirds

Thanks

**Comment by notshi** [comment]
> Before proceeding, just wanted to check with @notshi & @xriss that this statement ("Where does the money come from (IATI all years)") now relates to a breakdown of the relevant reporting-org ?

Yes. The donors page grouped by CRS but not this publishers page.

The phrases, "Where does the money come from / go?" were chosen to give context to the data for users; most will not know or care about reporting/participating orgs.

This is probably part of the (larger) discussion on d-portal's direction to be more publisher-focused (the IATI standard) and less user-focused (less jargon, more intuitive for the layperson).
